### ISSUE TITLE
crashing when map change

### ISSUE THREAD

**Issue by RedyRT** [Neyami/CSO-Weapons#6]
Hi again, it seems the server just crashing when map is changing with your addon
when im commenting this addon the crashing is just disappearing
[2024-02-19.log](https://github.com/Neyami/CSO-Weapons/files/14336046/2024-02-19.log)
Heres the log file, but 1 problem theres no any errors about it when it's just crashing 

Sadly i dont have any clue why its happening

**Comment by Neyami** [comment]
Yeah the game can be quite unstable like this and I don't know why either :/

Does it crash when changing to any level or just some? I've had the game crash when changing to stadium4 for instance, but not my test maps.

Could be that there are too many resources being loaded if you have too many weapons enabled, then add to that a map with a lot of entities already.  🤔

**Comment by RedyRT** [comment]
> Yeah the game can be quite unstable like this and I don't know why either :/
> 
> Does it crash when changing to any level or just some? I've had the game crash when changing to stadium4 for instance, but not my test maps.
> 
> Could be that there are too many resources being loaded if you have too many weapons enabled, then add to that a map with a lot of entities already. 🤔

i think its crashes to any level

**Comment by RedyRT** [comment]
i have insurgency weapons but i tried without them loading just CSO, seems same just crash

**Comment by Neyami** [comment]
Yeah the only thing I can think of atm. is to submit the crash dump or whatever they're called to the svencoop devs :/

And maybe disable one weapon at a time to see if it's one of them that causes it?

**Comment by RedyRT** [comment]
> Yeah the only thing I can think of atm. is to submit the crash dump or whatever they're called to the svencoop devs :/
> 
> And maybe disable one weapon at a time to see if it's one of them that causes it?

hmm, interesting idea, i can try i think

**Comment by RedyRT** [comment]
ERROR: Attempted to get current module while outside of a script!

thats what i get, and i think its with any weapon

**Comment by RedyRT** [comment]
i literally removed everything xd 
![изображение](https://github.com/Neyami/CSO-Weapons/assets/64935439/322f4cfb-498b-4dc1-8165-d21f75bac53e)
and it works fine just with 2 includes

**Comment by Neyami** [comment]
Try setting bUseDroppedItemEffect to false in csocommon.as
Maybe that effect has something to do with it? (it shouldn't)

**Comment by h266579110** [comment]
It's seen sometimes cause server frozed and timed out crash, i think is CBaseCSOWeapon cause problem (Every weapons are possible cause this problem.)
